pages/common/cloc.md

```diff
@@ -0,0 +1,15 @@
+# cloc
+
+> Count, and compute differences of, lines of source code and comments.
+
+- Count all the lines of code in a directory:
+
+`cloc {{directory}}`
```

Comment:
Sorry if I wasn't clear before. I meant we should leave this as just `{{/path/to/directory}}`.